src/services/Request.ts

```diff
@@ -0,0 +1,87 @@
+import * as _ from 'lodash'
+import fetch from 'node-fetch'
+import { ApiClientConfigurator } from '../configuration/ApiClientConfigurator'
+import { IConfiguration } from '../configuration/IConfiguration'
+
+export class Request {
+  protected config: IConfiguration
+  protected baseUrl: string = 'https://api.hubapi.com'
+  protected authTypes = {
+    apiKey: 'qs',
+    accessToken: 'oauth2',
+    developerApiKey: 'qs',
+  }
+
+  constructor(config: IConfiguration = {}) {
+    this.config = config
+  }
+
+  public async sent(opts: any) {
```

Comment:
I think you should extract this method from `Request` creating a separate `HttpClient` or similar class